pkg/bench/trace.go

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Tetragon
+
+package bench
+
+import (
+	"context"
+	"log"
+	"os"
+	"strings"
+	"time"
+)
+
+type TraceBench interface {
+	Crd(ctx context.Context, args *Arguments) string
```

Comment:
What is this method supposed to do? Maybe we can come up with a more informative name?

Based on the caller, it returns the configuration filename? Maybe `ConfigFilename()`?

Reply:
right, it gives back the CRD file path. will rename that